src/effects/native/phasereffect.cpp

```diff
@@ -0,0 +1,173 @@
+#include "util/math.h"
+#include <QDebug>
+
+#include "effects/native/phasereffect.h"
+
+const unsigned int updateCoef = 32;
+
+// static
+QString PhaserEffect::getId() {
+    return "org.mixxx.effects.phaser";
+}
+
+// static
+EffectManifest PhaserEffect::getManifest() {
+    EffectManifest manifest;
+    manifest.setId(getId());
+    manifest.setName(QObject::tr("Phaser"));
+    manifest.setAuthor("The Mixxx Team");
+    manifest.setVersion("1.0");
+    manifest.setDescription(QObject::tr(
+                "A more complex sound effect obtained by mixing the input signal" 
+                "with a copy passed through a series of all-pass filters."));
+    
+    EffectManifestParameter* stages = manifest.addParameter();
+    stages->setId("stages");
+    stages->setName(QObject::tr("Stages"));
+    stages->setDescription("Sets number of stages.");
+    stages->setControlHint(EffectManifestParameter::CONTROL_KNOB_LINEAR);
+    stages->setSemanticHint(EffectManifestParameter::SEMANTIC_UNKNOWN);
+    stages->setUnitsHint(EffectManifestParameter::UNITS_UNKNOWN);
+    stages->setDefault(1.0);
```

Comment:
The old version was correct. I think it is common to set the stages in double steps.
2 stages = 1 notch  
You find 2/6/12 stages phaser in the wild.